docs/index.md

```diff
@@ -0,0 +1,174 @@
+# Welcome to Saleor App Framework 
+
+You are reading the Saleor App Framework (Python) documentation. This document should help you to quickly bootstrap a 3rd Party Saleor App, read more about those [:saleor-saleor: Saleor's documentation](https://docs.saleor.io/docs/3.0/developer/extending/apps/key-concepts).
+
+The only supported web framework is **FastAPI**.
+
+## Quickstart
+
+### Install the framework
+
+Using Poetry (recommended):
+
+```bash
```

Comment:
Shouldn't be 👇  ?
```
pip install poetry
poetry add git+https://github.com/saleor/saleor-app-framework-python.git@main
```

Reply:
I'll link to this: https://python-poetry.org/docs/#installation as `pip install poetry` is not even the right thing to do